msfr/steady/msfr_ns.i

```diff
@@ -540,34 +655,49 @@ beta6 = 0.000184087
 []
 
 [Functions]
-  [mu_func]
+  [artificial_mu_func]
     type = PiecewiseLinear
     x = '1 2 3'
-    y = '${fparse 1e5*mu_t} ${fparse 1e2*mu_t} ${mu_t}'
+    y = '${mu} ${mu} ${mu}'
   []
 []
 
 [Materials]
   [mu]
-    type = ADGenericFunctionMaterial
-    prop_names = 'mu_t'
-    prop_values = 'mu_func'
+    type = ADGenericFunctionMaterial      #defines mu artificially for numerical convergence
+    prop_names = 'mu'                     #it converges to the real mu eventually.
+    prop_values = 'artificial_mu_func'
+  []
+  [total_viscosity]
+    type = MixingLengthTurbulentViscosityMaterial
+    u = 'v_x'                             #computes total viscosity = mu_t + mu
+    v = 'v_y'                             #property is called total_viscosity
+    mixing_length = mixing_len
+    mu = 'mu'
+    rho = ${rho}
     block = 'fuel pump hx'
   []
   [alpha]
     type = ADGenericConstantMaterial
-    prop_names = 'alpha'
-    prop_values = '${alpha}'
+    prop_names = 'alpha k cp  cp_unitary'  #cp_unitary defined for time kernel, to set rho_cp = 1
+    prop_values = '${alpha} ${k} ${cp} 1'
     block = 'fuel pump hx'
   []
   [ins_fv]
     type = INSFVMaterial
     u = 'v_x'
     v = 'v_y'
     pressure = 'pressure'
+    temperature = 'T'
     rho = ${rho}
     block = 'fuel pump hx'
   []
+  [not_used]
+    type = ADGenericConstantMaterial
+    prop_names = 'not_used'
+    prop_values = 0
+    block = 'shield reflector'
+  []
```

Comment:
delete the blocks
OR
use material_coverage_check = false

Reply:
actually do not set use material_coverage_check = false. dont want this to pop up in people s inputs more than it needs to